src/form-workflows.rst

```diff
@@ -0,0 +1,215 @@
+
+For example, a form definition could have a section for doing initial assessment of a dog suspected to have guinea worm, another section for selecting a treatment and carrying it out, another section for follow-up and a final section for certifying that the worm has exited. This whole workflow could be carried out by a single person over a series of days and then submitted at the end. 
+
+In the example above, the person would start with a new blank form when they first hear about the suspected guinea worm case. They would fill out intake information and then close the form without finalizing it. When they are ready for the next step of the workflow, they would go to :ref:`Edit Saved Forms <editing-saved-forms>` and use the :ref:`instance name <instance-name>` to find the appropriate submission to continue filling out. They could then use the :ref:`hierarchy view <jumping>` to find the section to fill out.
+
+When designing these kinds of forms, thoughtful :ref:`group names <groups>` and :ref:`instance names <instance-name>` can really help with navigation. Both can include status information that help data collectors take the appropriate next step.
+
+* :fa:`external-link` `Example of a single form supporting multiple encounters <https://docs.google.com/spreadsheets/d/1I9kWZOh4lbIR9T35iSC2YxLQeBI6ZApoBp_C7twx5Ec>`_
+
+There are limitations to representing multiple encounter workflows with a single form:
+
+* A single device must be used to perform the whole workflow.
+* There’s no opportunity to clean, monitor or report on data between steps of the workflow.
+* If the workflow needs to be interrupted, finding the correct filled form to edit and the correct question to jump to can be tedious (:ref:`instance name <instance-name>` and :ref:`groups <groups>` help with this).
+* A single long form can be hard to test and troubleshoot.
+
+.. _multiple-encounters-across-different-submissions:
+
+Multiple encounters across different submissions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Many workflows can’t be captured by a single form submission because:
+
+* Different steps of the workflow are completed by different people and/or at different times
+* Data is needed between steps for things like reports, monitoring, cleaning, assignment, eligibility determination
+
+Workflows can be split across multiple submissions of the same or different forms. The submissions are linked during analysis by a common entity identifier. The identifier can also optionally be used to :ref:`look up data about the selected entity <using-data-from-previous-encounters>`.
+
+The sections below describe some common tools and patterns for capturing multiple encounters across different submissions. As long as multiple submissions have common data, they can be linked no matter what their structure is so many variations are possible. 
```

Comment:
s/common/the same/

Reply:
As long as multiple submissions each have one field representing the same identifier.